libmproxy/script.py

```diff
@@ -4,6 +4,9 @@
 import threading
 import shlex
 import sys
+from watchdog.observers import Observer
+from watchdog.events import PatternMatchingEventHandler, FileModifiedEvent
+from .console import signals
```

Comment:
Unfortunately, we cannot use the "signals" from the `console` submodule here, which is specific to mitmproxy. Instead, we should add a log event to the master's queue (the "Channel" part in http://docs.mitmproxy.org/en/stable/dev/architecture.html) as it is done [here](https://github.com/mitmproxy/mitmproxy/blob/3739e1fe82c83ada3289dea5e24f9efcc7175696/libmproxy/proxy/root_context.py#L115). This is a bit tricky, I'm happy to take this over if you like.